app/Models/Campaign.php

```diff
@@ -41,6 +41,8 @@ class Campaign extends Model
     protected $casts = [
         'time_start' => 'string',
         'time_end' => 'string',
+        'targeting_requires' => 'json',
+        'targeting_excludes' => 'json',
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "targeting_excludes" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)